### ISSUE TITLE
`abs` and `aur` should be removed from archlinux plugin

### ISSUE THREAD

**Issue by jeffmhubbard** [ohmyzsh/ohmyzsh#8591]
I'm having some trouble with the `pacupd` alias now that I'm using aurutils. The usual output is followed by aurutil's usage text.

```
jeff@pythos ~
➜ % pacupd                                                                                            130 ↵ 16:01:11
:: Synchronizing package databases...
 testing                                   25.7 KiB  1116 KiB/s 00:00 [#######################################] 100%
 core                                     133.5 KiB  3.03 MiB/s 00:00 [#######################################] 100%
 extra                                   1640.3 KiB  17.8 MiB/s 00:00 [#######################################] 100%
 community-testing                        596.5 KiB  12.7 MiB/s 00:00 [#######################################] 100%
 community                                  4.8 MiB  17.5 MiB/s 00:00 [#######################################] 100%
 aur-builds is up to date
usage: aur [command]

available commands:
build		fetch		pkglist		repo		search		sync
depends		graph		query		repo-filter	srcver		vercmp

available user commands:
jeff@pythos ~
➜ %                                                                                                     1 ↵ 16:06:16
```

I've looked at the plugin, and found pacman and the AUR helpers all seem to have a similiar "update" alias, calling `sudo aur` and `sudo abs` if those commands are available.
```sh
if (( $+commands[abs] && $+commands[aur] )); then
  alias pacupd='sudo pacman -Sy && sudo abs && sudo aur'
elif (( $+commands[abs] )); then
  alias pacupd='sudo pacman -Sy && sudo abs'
elif (( $+commands[aur] )); then
  alias pacupd='sudo pacman -Sy && sudo aur'
else
  alias pacupd='sudo pacman -Sy'
fi
```

I think these checks for `aur` and `abs` may be dated and broken. Aurutils is the only package I can find providing `aur` and Arch Build System (`abs`?) was deprecated in 2017. This is only a problem after installing aurutils because it provides the `aur` command, but `sudo aur` is not the correct way to use it. I think `alias pacupd='sudo pacman -Sy'` is the only valid bit of that if statement left.

**Comment by mcornella** [comment]
Can you open a PR removing those so we can discuss it there? It should be easy now that the plugin was refactored.

**Comment by mcornella** [comment]
Fixed in #9803.
